mirror_maker.go

```diff
@@ -183,6 +215,22 @@ func (this *MirrorMaker) initializeMessageChannels() {
 }
 
 func (this *MirrorMaker) startProducers() {
+	if this.config.TimingsProducerConfig != "" {
+		conf, err := ProducerConfigFromFile(this.config.TimingsProducerConfig)
```

Comment:
Can you update the README on the main page with some more information about how to-do this please with a sample config, thanks!